go/integTest/glide_test_suite_test.go

```diff
@@ -190,3 +190,9 @@ func (suite *GlideTestSuite) verifyOK(result api.Result[string], err error) {
 	assert.Nil(suite.T(), err)
 	assert.Equal(suite.T(), api.OK, result.Value())
 }
+
+func (suite *GlideTestSuite) SkipIfServerVersionLowerThan7() {
```

Comment:
I propose to make this method parametrized. Some API is added in 6.2 or 8.0.

Reply:
@janhavigupta007 I added a new parameter for this function.